.github/workflows/ci.yml

```diff
@@ -212,6 +212,7 @@ jobs:
 
   # TODO: https://github.com/FuelLabs/fuel-indexer/issues/269
   cargo-test-workspace-all-features:
+    if: github.event_name != 'release' && github.event.action != 'published'
```

Comment:
If we're cutting a release then CI must've already passed (since release PR's only include version bumps), so we don't need to do this long CI step again